src/frontend/src/views/Data/tableHeaders/types.ts

```diff
@@ -0,0 +1,21 @@
+export interface SubHeader {
+  key: string;
+  text: string;
+}
+
+export type TooltipTextType = {
+  boldText: string;
+  regularText: string;
+  link?: {
```

Comment:
😭 I know we do this all over, but we really shouldn't have links in tooltips since they aren't reachable with the keyboard.

Reply:
Make a ticket in the eng backlog and maybe we can prioritize changing things!